lib/rules/no-extra-parens.js

```diff
@@ -386,6 +386,24 @@ module.exports = {
             return node && (node.type === "Identifier" || node.type === "MemberExpression");
         }
 
+        /**
+         * Checks if a node is fixable.
+         * A node is fixable if removing a single pair of surrounding parentheses does not turn it
+         * into a directive.
+         * @param {ASTNode} node The node to evaluate.
+         * @returns {boolean} Whether or not the node is fixable.
+         * @private
+         */
+        function isFixable(node) {
+            if (!(node.type === "Literal" && typeof node.value === "string")) {
```

Comment:
Can we refactor this a bit to make it easier to understand? And maybe add a comment?

```suggestion
            // if it's not a string literal it can be autofixed
            if (node.type !== "Literal" || typeof node.value !== "string")) {
```